dir.props

```diff
@@ -443,6 +445,6 @@
     <SkipTests Condition="'$(SkipTests)'=='' and ('$(OsEnvironment)'=='Windows_NT' and '$(TargetsWindows)'!='true' and '$(OSGroup)'!='AnyOS')">true</SkipTests>
   </PropertyGroup>
 
-  <Import Project="$(RoslynPropsFile)" Condition="'$(OsEnvironment)'=='Unix' and Exists('$(RoslynPropsFile)')" />
+  <Import Project="$(RoslynPropsFile)" Condition="('$(OsEnvironment)'=='Unix' or '$(OsEnvironment)'=='OSX') and Exists('$(RoslynPropsFile)')" />
```

Comment:
Even though we are still using existing msbuild we need to start using Roslyn as part of our windows builds as well. Can you please file an issue tracking that? I've been promising people that we can move to C# 6.0 after we get this work in.